ZenPacks/zenoss/ZenPackLib/lib/spec/GraphDefinitionSpec.py

```diff
@@ -21,7 +24,7 @@ def __init__(
             hasSummary=None,
             graphpoints=None,
             comments=None,
-            _source_location=None
+            _source_location=None,
```

Comment:
No I didn't b/c it kept failing when I did and worked when I didn't.  I just realized that the log=None parameter is already there a few lines up.  I'm setting the default like the others